src/bioclip/__main__.py

```diff
@@ -7,36 +7,41 @@
 
 Options:
   -h --help
-  --format=FORMAT      format of the output (table, json, or csv) [default: table]
+  --format=FORMAT      format of the output (table or csv) [default: csv]
   --rank=RANK          rank of the classification (kingdom, phylum, class, order, family, genus, species) [default: species] 
   --k=K                number of top predictions to show [default: 5]
   --cls=CLS            comma separated list of classes to predict, when specified the --rank and --k arguments are ignored [default: all]
   --output=OUTFILE     save output to a filename instead of printing it [default: stdout]
```

Comment:
```suggestion
  --output=OUTFILE     print output to file OUTFILE [default: stdout]
```
Matching change in README.